docs/modules/user-guide/partials/proc_limiting-resources-usage.adoc

```diff
@@ -30,88 +30,3 @@ This limit will be applied to every container of the given component.
 +
 For `plugin` components, RAM limits can be described in the plug-in descriptor file, typically named `meta.yaml`.
 +
-
-. Specify container memory request for components
-+
-To specify a container(s) memory request for `plugin` use the `memoryRequest` parameter:
-+
-.Specify container memory request for components
-====
-[source,yaml]
-----
-  components:
-   - name: exec-plugin
-     plugin:
```

Comment:
Although memory request stays, plugin will be removed from the devfile spec (https://github.com/devfile/api/pull/333) in the future. We can remove the plugin part of this example and leave the container part.

Reply:
The plugin is still there in the example, given that we plugin will be removed from the schema, we should remove all plugin definitions in this page.

Reply:
Okay, thanks. I removed the plugin references. I wasn't sure if I ought to have removed the id, memorylimit, and memoryrequest references under `name`.  I see int eh schema that `component/name` doesn't have these subparts, but still, didn't know if they needed to be removed if the plugin reference was removed.